### DISCUSSION TITLE
how to use "right click" as an option for clicking?

### DISCUSSION THREAD

**Discussion by ja77876** [noah-nuebling/mac-mouse-fix#315] [Questions] [answered]
I have an evoluent mouse. it has 3 main mouse buttons and I want to make the middle click mouse button do the "right click" function. However under the menu items under "middle button" it only gives the option of "middle click" not "right click." it also has options for many other things lick back, show desktop, launchpad, etc. But is there any way to have the option of "right click" added?

**Comment by noah-nuebling** [comment, marked as answer]
Hi @ja77876! This is currently not an option, but I'm thinking about it, and I've created Issue #357 to give your problem more visibility.

Let me know if you have more questions!

---

I'm marking this as answered to clean things up. Please continue discussion in #357. Thanks!
